lib/Importer.js

```diff
@@ -401,6 +401,11 @@ export default class Importer {
       return undefined;
     }
 
+    const distanceToCurrentFile = (importPath: string): number => {
+      const match = path.relative(this.pathToCurrentFile, importPath).match(/\//g);
```

Comment:
These paths should be normalized, so no need to use `path.sep`. I'll let you make the call on the `path.relative`. Either just remove it (and not worry about relative or absolute import paths), or keep it around for slightly better absolute path support. Let me know when you're done and I'll merge.